content/homepage.js

```diff
@@ -0,0 +1,40 @@
+import { FaDiscord, FaPlayCircle } from "react-icons/fa";
```

Comment:
This content file keeps our text OUT of our hero component

We could use a CMS for this, or MDX but for now this is probably fine.

We can also set this up to use custom components and import them dynamically via an array if you want really customisable pages. Or you can hardcode each component, up to you guys.